ui/src/app/utils/runtime-hooks.tsx

```diff
@@ -311,15 +318,6 @@ export const useCustomRuntime = (
           )
         );
 
-        // A disk update may be need in combination with a runtime update.
```

Comment:
why did this logic move to a different location?

Reply:
I was placing the updateRuntimes conditions together for making it easy to read, i can move it back

Reply:
Done